devscripts/changelog.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+from __future__ import unicode_literals
+
+import os
+import subprocess
+import sys
+
+
+def run(args):
+    process = subprocess.Popen(args, stdout=subprocess.PIPE, universal_newlines=True)
+    return process.communicate()[0].strip()
```

Comment:
Should this be using `utils.process_communicate_or_kill()`?

Reply:
While that would work, yeah, it would require me importing `youtube_dl`. Since the process is short running and well behaving (never had problems with `git` binary before) i think its not needed